.github/release-drafter.yml

```diff
@@ -7,3 +7,5 @@ template: |
 categories:
     - title: '🧶 Avhengighetsoppdateringer'
       label: 'dependencies'
+    - title: '🛠  Teknisk'
```

Comment:
Ville det vært pirkete å påstå at det er double-space her :p

Reply:
Git diffen lokalt hos meg viste 0 spaces, så jeg måtte legge inn to for at det skulle bli likt som den andre :(

Reply:
Unicode ass :]